slackbot-background/background.go

```diff
@@ -21,6 +22,8 @@ const (
 	WHITELIST_EMAIL_SLASH_COMMAND         = "/whitelist_email"
 	STAGING_WHITELIST_EMAIL_SLASH_COMMAND = "/staging_whitelist_email"
 
+	SECOPS_911_COMMAND = "/secops911"
```

Comment:
Might want a staging command as well. A slash command can only point to a specific endpoint, so the staging slash commands point to the staging slackbot and the regular ones point to the prod slackbot endpoint.